packages/astro/src/core/build/plugins/plugin-content.ts

```diff
@@ -39,40 +41,44 @@ interface ContentManifest {
 	// Tracks components that should be passed to the client build
 	// When the cache is restored, these might no longer be referenced
 	clientEntries: string[];
+	// Hash of the lockfiles, pnpm-lock.yaml, package-lock.json, etc.
+	// Kept so that installing new packages results in a full rebuild.
+	lockfiles: string;
+	// Hash of the Astro config. Changing options results in invalidating the cache.
+	configs: string;
```

Comment:
nit: he name of the property is plural, but it's only one string. Should it be `config` instead?

Reply:
I went with `configs` because we might add other config checks in the future (like Tailwind config).

Reply:
Oh I see, thank you